container_linux.go

```diff
@@ -528,13 +556,6 @@ func (c *linuxContainer) criuSwrk(process *Process, req *criurpc.CriuReq, opts *
 		log.Warn(st.String())
 	}()
 
-	if process != nil {
```

Comment:
@avagin It's not clear to me if this is actually needed, but if it is I will put it back.